src/components/tooltip/tooltip.tsx

```diff
@@ -153,13 +153,15 @@ export class Tooltip {
     private addListeners() {
         this.ownerElement?.addEventListener('mouseover', this.showTooltip);
         this.ownerElement?.addEventListener('mouseout', this.hideTooltip);
-        this.ownerElement?.addEventListener('click', this.hideTooltip);
```

Comment:
The `click` event listener was removed because it might interfere with the desired behavior of showing the tooltip on `focus` and hiding it on `blur`.

If a user clicks on the trigger element, it will also receive `focus`. If the tooltip is set to hide on `click`, it would immediately disappear after appearing, which is probably not what we want.